tools/cell_census_builder/__main__.py

```diff
@@ -313,8 +313,12 @@ def create_args_parser() -> argparse.ArgumentParser:
         default=True,
         help="Consolidate TileDB objects after build",
     )
-    # hidden option for testing. Will process only the first 'n' datasets
+    # hidden option for testing by devs. Will process only the first 'n' datasets
     build_parser.add_argument("--test-first-n", type=int, help=argparse.SUPPRESS)
+    # hidden option for testing by devs. Allow for WIP testing by devs.
+    build_parser.add_argument(
+        "--test-disable-dirty-git-check", action=argparse.BooleanOptionalAction, help=argparse.SUPPRESS
```

Comment:
I'd prefer we don't hide the option. It's just "us" reading the --help.

Reply:
I assume you feel the same with the other --test option?

Reply:
yes--didn't realize it was also suppressed.